plugins/net_plugin/net_plugin.cpp

```diff
@@ -3013,6 +3016,13 @@ namespace eosio {
             }
          }
 
+         if( options.count( "private-peer" )) {
+            const auto& private_peers = options.at("private-peer").as<std::vector<std::string>>();
```

Comment:
I correct understand, that `private-peer` should be specified in `<host>:<port>` format?

Reply:
@s-medvedev Yes.